README.md

```diff
@@ -1 +1,22 @@
-# cri-containerd
\ No newline at end of file
+# cri-containerd
+Cri-containerd is a [containerd](https://containerd.io/) based implementation of Kubernetes [container runtime interface (CRI)](https://github.com/kubernetes/kubernetes/blob/v1.6.0/pkg/kubelet/api/v1alpha1/runtime/api.proto).
```

Comment:
Hm, based on this https://english.stackexchange.com/questions/9063/how-do-you-capitalize-a-proper-noun-such-as-iphone, it should be "cri-containerd" here. Will update.